sickbeard/webapi.py

```diff
@@ -254,6 +254,8 @@ def showPoster(self, show=None, which=None):
         # Redirect initial poster/banner thumb to default images
         if which[0:6] == 'poster':
             default_image_name = 'poster.png'
+        elif which[0:6] == 'fanart':
```

Comment:
I added this from `webserve.py`. I'm not sure if it is require in the API or not.